### PR TITLE
give us the S U C C

### PR DESCRIPTION
**Pull request by JordanReimers**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -160,6 +160,9 @@ bot.addListener("message", function(from, to, text, message) {
 		}
 		else bot.action(config.channels[0], "dabs on "+from +" "+randomFromArray(dabEmojis));
 	}
+	if((message.args[1].toLowerCase().indexOf("succ") > -1){
+		bot.say(config.channels[0], "S U C C");
```
Comment (chris30269) [inline main.js]: are you making a command, or a response to a trigger word?

Comment (JordanReimers) [reply main.js]: response to a trigger word

Comment (chris30269) [reply main.js]: @JordanReimers pls

Comment (JordanReimers) [reply main.js]: approve me you coward

**Review by chris30269 — APPROVED** [review]

**Pull request merged**